source/js/components/Confirmation/Confirmation.tsx

```diff
@@ -1,21 +1,19 @@
-import { formatTimePeriod } from '../helpers/BookingHelper';
+import { formatTimePeriod } from '../../helpers/BookingHelper';
 
 export interface ConfirmationInterface {
   administratorName: string;
   userEmail: string;
-  date: string;
-  startTime: string;
-  endTime: string;
+  date?: string;
+  startTime?: string;
+  endTime?: string;
 }
-const Confirmation = ({ administratorName, userEmail, date, startTime, endTime }: ConfirmationInterface) => (
+export const Confirmation = ({ administratorName, userEmail, date, startTime, endTime }: ConfirmationInterface) => (
```

Comment:
consider default the import params instead:

```js
export const Confirmation = ({ administratorName, userEmail, date = '', startTime = '', endTime = '' }: ConfirmationInterface)
```

by doing that we don't need to have logic like:
```js
date ?? ''
```
which means that our function calls gets a little bit cleaner:

```js
formatTimePeriod(date , startTime, endTime)
```

Reply:
I agree. Fixed